test/test_transform.py

```diff
@@ -1262,18 +1262,12 @@ def test_privatize_with_nonzero_lbound(ctx_factory):
 
 def test_simplify_indices(ctx_factory):
     ctx = ctx_factory()
-    twice = lp.make_function(
-        "{[i, j]: 0<=i<10 and 0<=j<4}",
-        """
-        y[i,j] = 2*x[i,j]
-        """, name="zerozerozeroonezeroify")
-
```

Comment:
How come you got rid of the inlining version of the test? (Also, related: what did the inlining version have to do with index simplification? @kaushikcfd?)

Reply:
Checking for unsimplified expressions after inling a kernel was unnecessary, so I was fine if we didn't require it in the tests.